src/components/ResultViewer.js

```diff
@@ -23,7 +24,12 @@ export class ResultViewer extends React.Component {
   static propTypes = {
     value: PropTypes.string,
     editorTheme: PropTypes.string,
+    ResultsTooltip: PropTypes.any,
   };
+  constructor() {
+    super();
+    this._tooltip = document.createElement('div');
```

Comment:
The DOM creation could happen when `componentDidMount` is called. I have a separate (but similar with yours in nature) concern with creating a class variable that would just act as a placeholder.